interview-questions.md

```diff
@@ -7,50 +7,64 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
+  Your answer: 
 
   Researched answer:
 
 
-
-1. What is git? What is the difference between git and Github?
+1. What is git? What is the difference between git and Github? *
 
   Your answer:
 
+    -The difference between Git and Github is that Git is local and Github is remote. Git is a software where you interact with your own personal computer. Github is a cloud like platform where you make changes online.
+
   Researched answer:
 
+    -Through my research I learned that the difference between Git and Github is indeed the fact that Git is local and Github is remote. To further go on detail, Git is a graphical user interface keeping track of your past actions retaining the source code during the software development process through your own local personal computer. Github in the other hand is a way you store your Git files via online, a service focused on centralizing source code with places in which codes are stored to a directory and essentialy a hosting service that can be available or accesed to the public allowing you to share your Git and other Developers are able to pair program or work/alter on your Git codes remotely.
 
 
-2. Which JavaScript operators will return a Boolean value?
+2. Which JavaScript operators will return a Boolean value? *
 
-  Your answer:
+  Your answer: 
 
-  Researched answer:
+    -In my brief experience learning how to code this week. I remember that the JavaScript operator that will return a Boolean value is a Relational operator.
```

Comment:
Yup correct. Relational operators will return a Boolean value as will equality operators and logical operators.